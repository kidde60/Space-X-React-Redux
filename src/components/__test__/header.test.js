import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Header from '../header/header';

it('<Header /> renders properly as expected', () => {
  const tree = renderer.create(<BrowserRouter><Header /></BrowserRouter>).toJSON();
  expect(tree).toMatchSnapshot();
});
