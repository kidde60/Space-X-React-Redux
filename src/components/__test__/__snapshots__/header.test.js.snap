// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders properly as expected 1`] = `
<nav>
  <header>
    <div
      className="navbar"
    >
      <div>
        <img
          alt="Space X logo"
          className="logo"
          src="planet.png"
        />
      </div>
      <ul
        className="navList"
      >
        <li
          className="navLink"
        >
          <a
            aria-current="page"
            className="active"
            href="/"
            onClick={[Function]}
          >
            Rokets
          </a>
        </li>
        <li
          className="navLink"
        >
          <a
            className=""
            href="/missions"
            onClick={[Function]}
          >
            Missions
          </a>
        </li>
        <li
          className="navLink"
        >
          <a
            className=""
            href="/profile"
            onClick={[Function]}
          >
            Profile
          </a>
        </li>
      </ul>
    </div>
  </header>
</nav>
`;
